src/controllers/users.js

```diff
@@ -0,0 +1,52 @@
+import UserModel from "../models/users";
+import jwt from "jsonwebtoken";
+import bcrypt from "bcrypt";
+import { validateBody } from '../utils/index'
+
+export async function registerUser(req, res) {
+  try {
+    const { user } = req.body;
+    const expectedPayload = {
+      username: 'string',
+      email: 'string',
+      password: 'string'
+    }
+    if(!user){
+      res.status(400).send("Invalid request payload")
```

Comment:
Duplicated code here, Lets change validateBody to also check if body exists. It can also be used to return what the error is so a more detailed error is sent to the user.